api/README.md

```diff
@@ -0,0 +1,54 @@
+# SovBot API
+
+# Environment variables
+
+<table>
+    <tr>
+        <th>Variable name</th>
+        <th>Description</th>
+        <th>Default value</th>
+    </tr>
+    <tr>
+        <td>MONGODB_HOST</td>
+        <td>Full path to MongoDB. For example <code>mongodb://localhost:27017/dbname</code>.</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>JWT_SECRET</td>
+        <td>JSON Web Token secret.</td>
+        <td></td>
+    </tr>
+</table>
+
+# Routes
+
+## Account
+
+### POST `/account/join`
+Registration of new user
+
+JSON data:
+* username {string}
+* password {string}
+
+Response:
+```
+{
+  "username": "YOUR_USERNAME"
+}
+```
+
+### POST `/account/auth`
+Authorisation with existing user
+
+JSON data:
+* username {string}
+* password {string}
+
+Response:
+```
+{
+  "username": "YOUR_USERNAME",
```

Comment:
i think it's unnecessary. Best way to add swagger, and create a JSON to describe all routes.

Reply:
for this purpose swagger-ui-express is enough.